Dockerfile.tests

```diff
@@ -13,8 +13,8 @@ RUN apk --update add make openssl python \
 #  - v8.0.0027 (Vim 8, real async support)
 #  - v8.0.8069 (Updated Vim 8, https://groups.google.com/forum/#!topic/vim_dev/ro_FpeXgc24)
 RUN install_vim \
-        -tag v7.3.429 -name vim73 -build \
-        -tag v7.4.052 -name vim74-trusty -build \
+        -tag v7.3.429 -py -name vim73 -build \
+        -tag v7.4.052 -py -name vim74-trusty -build \
```

Comment:
I'm quite sure vim on trusty has python support and for vim 73 I don't know.

But maybe it's better to just ignore these tests for old versions of vim. I'm not sure how to do that though. If you could change that it would be great.